src/components/ChatsHeader/ChatsHeader.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <header class="unnnic-chats-header" :class="{ large: size === 'large' && !avatarName }">
+  <header class="unnnic-chats-header" :class="{ large: size === 'large', contact: !!avatarName }">
```

Comment:
I don't know if it is the case but you can write `class="[size, { contact: !!avatarName }]"`

Reply:
Really, it makes sense. Thanks!